android/app/src/main/java/org/uwaterloo/subletr/pages/chat/ChatListingPageView.kt

```diff
@@ -0,0 +1,160 @@
+package org.uwaterloo.subletr.pages.chat
+
+import androidx.compose.foundation.background
+import androidx.compose.foundation.layout.Arrangement
+import androidx.compose.foundation.layout.Box
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.height
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.layout.width
+import androidx.compose.foundation.lazy.LazyColumn
+import androidx.compose.foundation.shape.CircleShape
+import androidx.compose.foundation.shape.RoundedCornerShape
+import androidx.compose.material3.Surface
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.draw.clip
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.res.dimensionResource
+import androidx.compose.ui.text.font.FontWeight
+import androidx.compose.ui.unit.dp
+import androidx.compose.ui.unit.sp
+import androidx.hilt.navigation.compose.hiltViewModel
+import org.uwaterloo.subletr.R
+import org.uwaterloo.subletr.theme.subletrPink
+
+@Composable
+fun ChatListingPageView(
+	modifier: Modifier = Modifier,
+	viewModel : ChatListingPageViewModel = hiltViewModel()
+){
+	Column {
```

Comment:
Please add support for `Loading` uiState.
Also, I'm pretty sure that this should be a `LazyColumn`.